portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/questions/QuestionsWidgetActions.testcase

```diff
@@ -418,6 +418,66 @@ definition {
 			locator1 = "Questions#QUESTIONS_LIST_TAB_FIELD");
 	}
 
+	@description = "Validate if the users can view the questions from different topics via the breadcrumb."
+	@priority = "4"
+	test ViewQuestionsFromDifferentSubTopics {
+		JSONMBMessage.addCategory(
+			categoryDescription = "Description",
+			categoryName = "Questions Test Category",
+			userEmailAddress = "test@liferay.com");
+
+		Navigator.gotoPage(pageName = "Questions Page");
+
+		Questions.gotoCategory(categoryName = "Questions Test Category");
+
+		MessageBoardsAdmin.openMessageBoardsAdmin(siteURLKey = "guest");
+
+		MessageboardsCategory.gotoCP(categoryName = "Questions Test Category");
+
+		var messageBoardsCategoryURL = selenium.getLocation();
+		var parentCategoryId = StringUtil.extractLast("${messageBoardsCategoryURL}", "mbCategoryId=");
```

Comment:
If there is no space between these two lines then they can be done in any order. But because ``parentCategoryId`` depends on ``messageBoardsCategoryURL`` they need to go in order and we need to add a space between these lines

Reply:
Yeah, you are right, we need to get the location URL first, then go to get the category ID from the location URL. do you need I go to update it?